.github/workflows/docs-to-gh-wiki.yml

```diff
@@ -1,21 +0,0 @@
-name: Deploy Docs to GitHub Wiki
```

Comment:
Whoops, this wasn't meant to be deleted in this PR. Was doing some testing in my fork and this action was causing some build failures. Will add back